TLPhotoPicker/Classes/TLPhotosPickerViewController.swift

```diff
@@ -1342,17 +1347,21 @@ extension Array where Element == PopupConfigure {
 
 extension UIImage {
     public func colorMask(color:UIColor) -> UIImage {
```

Comment:
This works for what it's used for : the arrow that appears below "Tap here to change " when selecting a photo for a nutrition event